resources/views/auth/register.blade.php

```diff
@@ -30,16 +37,15 @@
 
         <div class="mb-3 has-feedback{{ $errors->has('name') ? ' has-error' : '' }}">
             <div class="flex items-stretch w-full">
-                <input id="name" type="text" class="rounded border p-2 w-full dark:bg-slate-800 dark:border-slate-500" name="name" value="{!! old('name') !!}" placeholder="{{ __('auth.register.fields.name') }}" required autofocus>
+                <input id="name" type="text" class="rounded border p-2 w-full dark:bg-slate-800 dark:border-slate-500" name="name" @if (app()->isLocal() && config('auth.fast_registration')) value="{{ $name }}" @endif placeholder="{{ __('auth.register.fields.name') }}" required autofocus>
```

Comment:
You need to keep old or someone who has an error in the form has to retype their data :)